src/api/graphql-client.ts

```diff
@@ -168,13 +169,14 @@ export const me = async (graphqlApiUrl: string, accessToken: string): Promise<Ex
         user {
           id
           username
+          avatar
           email
           walletAddress
         }
       }
     `,
     variables: { data: { accessToken } },
-    dataField: 'me',
+    dataField: 'user',
```

Comment:
At the moment the dataField's set as the `operationName`, for any of the remaining queries. Are you sure you need to change it?

```suggestion
    dataField: 'me',
```